torch/onnx/symbolic_opset9.py

```diff
@@ -6629,3 +6630,23 @@ def onnx_placeholder(g: jit_utils.GraphContext, *inputs, **attrs):
     env = g.env
 
     return torch._C._jit_onnx_convert_pattern_from_subblock(block, node, env)
+
+
+@_onnx_symbolic("aten::_conj")
+@_onnx_symbolic("aten::conj_physical")
+@_onnx_symbolic("aten::resolve_conj")
+@_onnx_symbolic("aten::resolve_neg")
+@_beartype.beartype
+def unsupported_complex_operators(g: jit_utils.GraphContext, input: _C.Value):
```

Comment:
Since `unsupported_complex_operators` is not an aten function, is it unlikely we want to call it from other symbolics?If so I would hide it  (Private by default, public only when needed):

```suggestion
def _unsupported_complex_operators(g: jit_utils.GraphContext, input: _C.Value):
```

Reply:
They might be called by any other symbolic that does not support complex. Renamed anyway